x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -108,7 +108,7 @@ const assertFilterControlsWithFilterObject = (
 };
 
 // FLAKY: https://github.com/elastic/kibana/issues/171890
```

Comment:
don't forget to remove the flaky comment

Reply:
yes.. just waiting on all Flaky runs to be completed.